third_party/com_github_pugixml/bundled.BUILD.bazel

```diff
@@ -0,0 +1,11 @@
+licenses(["notice"])
```

Comment:
I'd prefer that this was named `pugixml.BUILD.bazel`; I think that's more discoverable than `bundled.BUILD.bazel`, though I haven't renamed the older BUILD files.

Reply:
updated